python_on_whales/components/container/cli_wrapper.py

```diff
@@ -860,6 +861,9 @@ def execute(
                 to allow communication with the parent process.
                 Currently only works with `tty=True` for interactive use
                 on the terminal.
+            preserve_fds: The number of additional file descriptors to pass
+                through to the container. Only supported by podman.
+
```

Comment:
```suggestion
                through to the container. Only supported by podman.
```

Reply:
There is an empty extra line, this will break the docs formatting

Reply:
oops